### PR TITLE
Cleans up generated code output.

### PR DESCRIPTION
**Pull request by kdschlosser**
This is a pretty involved PR in what it does. There is a decent amount of code changes but nothing to crazy. Here are the things the PR does

removes duplicate imports
removes unused imports
puts declarations after imports instead of being mixed in with the imports
formats most code that is over 80 characters on a single line
2 blank lines before and after classes
puts class docstrings as the first line in the class  this way it can actually get set to `__doc__`
fixes lack of triple double quotes not being used around class docstrings

So in a nutshell it makes most of the generated code PEP8 compliant and it makes the generated code a whole hell of a lot easier on the :eyes:

### CODE REVIEW

**Comment by vasily-v-ryabov** [comment]
pywinauto tests with this PR branch: https://ci.appveyor.com/project/pywinauto/pywinauto-8b7t7/builds/42558727

One Python 3.6 line fails expectedly. The rest should pass. Hopefully.

**Review by vasily-v-ryabov — APPROVED** [review]
I can say it's good enough after quick look. If pywinauto tests are not broken, I will merge it.
